account_statement_base/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+from odoo.addons.web.controllers.utils import clean_action
+
+
+class AccountBankStatementLine(models.Model):
+
+    _inherit = "account.bank.statement.line"
+
+    def action_open_journal_entry(self):
+        self.ensure_one()
+        if not self:
+            return {}
+        action = self.env.ref("account.action_move_line_form").sudo().read()[0]
+        action["view_mode"] = "form"
+        action["res_id"] = self.move_id.id
+        del action["view_id"]
+        del action["views"]
```

Comment:
You are removing here the possible specific form view. This can have side effects.